tests/test_analysis.py

```diff
@@ -0,0 +1,87 @@
+import covasim as cv
+import unittest
+import numpy as np
+
+
+# Suite of tests to test basic functionality of the analysis.py file
+def test_analysis_snapshot():
+    sim = cv.Sim(analyzers=cv.snapshot('2020-04-04', '2020-04-14'))
+    sim.run()
+    snapshot = sim['analyzers'][0]
+    people = snapshot.snapshots[0]             # Option 1
+    people2 = snapshot.snapshots['2020-04-04'] # Option 2
+    people3 = snapshot.get('2020-04-14')       # Option 3
+    people4 = snapshot.get(34)                 # Option 4
+    people5 = snapshot.get()                   # Option 5
+    # people3 = []  # uncomment to verify error
+    i = 0
+    peoples = [people, people2, people3, people4, people5]
+    for p in peoples:
+        i+=1
+        if len(p) == 0:
+            raise ValueError("Option {} is not getting people array correctly".format(i))

```

Comment:
Please use an assertion here instead of just raising the ValueError.

Suggested something like this:
```
for i in range(len(peoples)):
    assert len(peoples[i]) > 0, msg=f"Option {i} should have more than 0 members.")
```

Reply:
Would also suggest `for i,p in enumerate(peoples)` rather than the increment